src/uct/ib/rc/base/rc_ep.h

```diff
@@ -549,8 +549,7 @@ uct_rc_ep_enable_flush_remote(uct_rc_ep_t *ep)
 
 static UCS_F_ALWAYS_INLINE int uct_rc_ep_is_flush_remote(uct_rc_ep_t *ep)
 {
-    return (ep->flags & UCT_RC_EP_FLAG_FLUSH_REMOTE) &&
-           uct_ib_md_is_flush_rkey_valid(ep->flush_rkey);
+    return ep->flags & UCT_RC_EP_FLAG_FLUSH_REMOTE;
```

Comment:
do we still need the function uct_rc_ep_is_flush_remote ? it's just checking a flag

Reply:
> do we still need the function uct_rc_ep_is_flush_remote ? it's just checking a flag

common code for rc verbs/mlx5. removed